content/docs/dvclive/api-reference/live/log_plot.md

```diff
@@ -0,0 +1,150 @@
+and stores the outputs at `{path}/plots/calibratrion.json` in a format
+compatible with `dvc plots`.
+
+```py
+y_true = [0, 0, 1, 1]
+y_score = [0.1, 0.4, 0.35, 0.8]]
+live.log_plot("calibration", y_true, y_score)
+```
+
+![](/img/dvclive-calibration.png)
+
+### `confusion_matrix`
+
+Generates a [confusion matrix](https://en.wikipedia.org/wiki/Confusion_matrix)
+plot.
+
+Stores the abels and predictions in `{path}/plots/confusion_matrix.json`, with
+the format expected by the confusion matrix
+[template](/doc/command-reference/plots#plot-templates-data-series-only) of
+`dvc plots`.
+
+```py
+y_true = [1, 1, 2, 2]
+y_pred = [2, 1, 1, 2]
+live.log_plot("confusion_matrix", y_true, y_pred)
+```
+
+![](/img/dvclive-confusion_matrix.png)
+
+### `det`
+
+Generates a
+[detection error tradeoff (DET)](https://scikit-learn.org/stable/modules/model_evaluation.html#det-curve)
+plot.
+
+Calls
+[sklearn.metrics.det_curve](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.det_curve.html)
+and stores the outputs at `{path}/plots/det.json` in a format compatible with
+`dvc plots`.
+
+```py
+y_true = [1, 1, 2, 2]
+y_score = [0.1, 0.4, 0.35, 0.8]
+live.log_plot("det", y_true, y_score)
+```
+
+![](/img/dvclive-det.png)
+
+### `precision_recall`
+
+Generates a
+[precision-recall curve](https://scikit-learn.org/stable/modules/model_evaluation.html#precision-recall-f-measure-metrics)
+plot.
+
+Calls
+[sklearn.metrics.precision_recall_curve](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.precision_recall_curve.html)
+and stores the outputs at `{path}/plots/precision_recall.json` in a format
+compatible with `dvc plots`.
+
+```py
+y_true = [1, 1, 2, 2]
+y_score = [0.1, 0.4, 0.35, 0.8]
+live.log_plot("precision_recall", y_true, y_score)
+```
+
+![](/img/dvclive-precision_recall.png)
+
+### `roc`
+
+Generates a
+[receiver operating characteristic (ROC) curve](https://scikit-learn.org/stable/modules/model_evaluation.html#roc-metrics)
+plot.
+
+Calls
+[sklearn.metrics.roc_curve](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_curve.html#sklearn.metrics.roc_curve)
+and stores the outputs at `{path}/plots/roc.json` in a format compatible with
+`dvc plots`.
+
+```py
+y_true = [1, 1, 2, 2]
+y_score = [0.1, 0.4, 0.35, 0.8]
+live.log_plot("roc", y_true, y_score)
+```
+
+![](/img/dvclive-roc.png)
+
+## Parameters
+
+- `name` - One of the supported plots.
+
+- `labels` - Array of ground truth labels.
+
+- `predictions` - Array of predicted labels (for `confusion_matrix`) or
+  predicted probabilities (for the rest of the supported plots)
+
+- `**kwargs` - Additional arguments to be passed to the internal scikit-learn
+  function being called.
+
+## Exceptions
+
+- `dvclive.error.InvalidPlotTypeError` - thrown if the provided `name` does not
+  correspond to any of the supported plots.
+
+- `RuntimeError` - thrown if `Live.log_plot()` is used alongside
+  `Live.next_step()` or `Live.set_step()`.
```

Comment:
This description should probably also be updated per the comment above about step behavior for `log_plot`.